client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-page-list.tsx

```diff
@@ -56,6 +56,9 @@ const PatternPageList = ( {
 	const categoriesInOrder = useCategoriesOrder( categories, ORDERED_PATTERN_PAGES_CATEGORIES );
 	const composite = useCompositeState( { orientation: 'vertical' } );
 
+	const getCategoryLabel = ( category: Category ) =>
+		category.name === 'posts' ? translate( 'Blog' ) : category.label;
```

Comment:
I think this check can get weird when we add more pages.

What if we just pick the title of the first (only) pattern in each category? Something like this:

```diff
@@ -86,6 +86,8 @@ const PatternPageList = ( {
                                                return null;
                                        }

+                                       const title = pagesMapByCategory[ name ][ 0 ].title;
+
                                        return (
                                                <CompositeItem
                                                        { ...composite }
@@ -95,10 +97,7 @@ const PatternPageList = ( {
                                                        aria-checked={ isSelected }
                                                        onClick={ () => onSelectPage( name ) }
                                                >
-                                                       <PatternPageListItem
-                                                               label={ getCategoryLabel( category ) }
-                                                               isSelected={ isSelected }
-                                                       />
+                                                       <PatternPageListItem label={ title } isSelected={ isSelected } />
                                                </CompositeItem>
                                        );
                                } ) }
```

Reply:
It feels odd to depend on the pattern name instead of the category name, but I agree that it's better for consistency 👍

Reply:
Yeah, I think we can revisit this later when we can allow more than 1 pattern per category 😄